doc/submitter_guide.md

```diff
@@ -60,12 +60,13 @@ dashboard](https://kcidb.kernelci.org/).
 2\. Install KCIDB
 -----------------
 
-Pick a KCIDB [release][releases]. Latest would normally be best.
 
-Let's say you picked v9. Run this:
+KCIDB employs continuous integration and delivery, and aims to keep the code working at all times.
+
+Please install the latest version from GitHub:
 
 ```bash
-pip3 install --user 'git+https://git@github.com/kernelci/kcidb.git@v9'
+pip3 install --user 'git+https://git@github.com/kernelci/kcidb.git@main'
```

Comment:
Could you please remove `@main` from the end? It will work without it, would look simpler, and wouldn't tie us to the particular name.

Reply:
Done, but I am a bit confused will it work?

Reply:
> Done, but I am a bit confused will it work?

It should, IIRC, the remote repository provides the default branch name to check out, and that's what pip uses. Regardless, you could always try it yourself :grin: